src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -123,6 +146,48 @@ public static void removeBiome(BiomeType type, BiomeEntry entry)
         return list != null ? ImmutableList.copyOf(list) : null;
     }
 
+    @Nullable
+    public static ImmutableList<OceanBiomeEntry> getOceanBiomes(OceanType type)
```

Comment:
Not threadsafe, missing documentation.

Reply:
Yes, this definitely a valid suggestion and I'll get around to fixing it 👍

This is also my response to the similar suggestions regarding thread safety and lack of documentation below.